game-of-agents.py

```diff
@@ -0,0 +1,83 @@
+﻿from envs import *
+from agents import *
+
+
+def inrow_heuristic(env, player, weight=0):
+    """
+    Score system:
+        win = inf
+        loss = -inf
+        array of some symbols without space to extend = 0
+        array with 1 symbol = +/-1 = +/-(1*1)
+        array with 2 symbols = +/-4 = +/-(2*2)
+        array with 3 symbols = +/-9 = +/-(3*3)
+    This scores growth if the array have multiply space to extend to!!!
+    """
+    # Check terminal state
+    if env.is_terminal_state():
+        if env.available_moves(player) > 0:
+            return float("inf")
+        elif env.available_moves(player) < 0:
+            return -float("inf")
+        else:
+            return 0
+
+    score = 0
+    directions = {"hor": (0, 1),
+                  "ver": (1, 0),
+                  "diag": (1, -1),
+                  "re-diag": (1, 1),
+                  }
+
+    # Run on every empty cell
+    for i in range(env.boardH):
+        for j in range(env.boardW):
+            if env.board[i][j] == 0:
+
+                # Run in every direction
+                for direc in directions:
+
+                    # Function to count len of symbols array that we have in some direction
```

Comment:
Please don't make the code look inflated... move this definition outside the function, or at least outside the loops.

Reply:
Give a little more detail